cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -112,7 +120,7 @@ public class KafkaBridgeCluster extends AbstractModel {
     protected static final String ENV_VAR_KAFKA_BRIDGE_CORS_ALLOWED_METHODS = "KAFKA_BRIDGE_CORS_ALLOWED_METHODS";
 
     protected static final String CO_ENV_VAR_CUSTOM_BRIDGE_POD_LABELS = "STRIMZI_CUSTOM_KAFKA_BRIDGE_LABELS";
-
+    protected static final String INIT_VOLUME_MOUNT = "/opt/strimzi/init";
```

Comment:
As mentioned in the commit message I thought it was better to mount it under the STRIMZI_HOME in the bridge container, where Connect and Mirrormaker put it in `/opt/kafka`

Reply:
I think that is fine. It makes sense here the way you did it 👍